apps/backend/src/db/migrations/20240821161830_add_quiz_questions_data.ts

```diff
@@ -0,0 +1,134 @@
+import { type Knex } from "knex";
+
+type Category = {
+	id: number;
+	name: string;
+};
+
+type QuizQuestion = {
+	[ColumnName.CATEGORY_ID]: number;
+	[ColumnName.LABEL]: string;
+};
+
+const TableName = {
+	CATEGORIES: "categories",
+	QUIZ_QUESTIONS: "quiz_questions",
+} as const;
+
+const Categories = {
```

Comment:
```suggestion
const Category = {
```

Reply:
'Category' is already defined as type Category. I've changed `const Category = {` to more specific `const CategoryName = {`